manifests/m/MaxBQb/BulkStartStop/1.0.2/MaxBQb.BulkStartStop.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MaxBQb.BulkStartStop
+PackageVersion: 1.0.2
+Installers:
+- InstallerUrl: https://github.com/MBQbUtils/BulkStartStop/releases/latest/download/BulkStartStop_setup.exe

```

Comment:
Please use the URL to the version and not to Latest, or else SHA mismatches will occur

Reply:
fixed